lib/prerequisite.js

```diff
@@ -9,6 +9,60 @@ module.exports = (input, pkg, opts) => {
 
 	const tasks = [
 		{
+			title: 'Ping npm registry',
+			task: () => new Promise((resolve, reject) => {
+				// `npm ping` waits wayyyy too long
+				const timeout = setTimeout(() => {
+					reject(new Error('Connection to npm registry timed out'));
+				}, 6000);
+				execa.stdout('npm', ['ping'])
+					.then(() => {
+						clearTimeout(timeout);
+						resolve();
+					}, () => {
```

Comment:
`.catch()`

Reply:
Patch coming in next commit. 

But for the sake of `arg`, is using `.catch` a decision of verbosity? convention? quirk? performance?

Reply:
Clarity, since they have subtle differences depending on how it's used.